source/opt/ir_context.h

```diff
@@ -25,10 +25,20 @@ namespace ir {
 
 class IRContext {
  public:
+  // Available analyses.
+  //
+  // When adding a new analysis:
+  //
+  // 1. Make sure it gets invalidated by IRContext methods that add or remove IR
+  //    elements (e.g., KillDef, KillInst, ReplaceAllUsesWith).
+  //
+  // 2. Add handling code in BuildInvalidAnalyses and
+  //    InvalidateAnalysesExceptFor.
   enum Analysis {
     kAnalysisNone = 0x0,
     kAnalysisBegin = 0x1,
     kAnalysisDefUse = kAnalysisBegin,
+    kAnalysisInstrToBlockMapping,
     kAnalysisEnd = 0x2
```

Comment:
This is suppose to be 1 past the end, so it should become 0x4.

Reply:
Done.